styles/_show.styl

```diff
@@ -102,4 +102,19 @@
     color lighten(black, 20%)
     border-bottom 1px solid yellow
     text-decoration none
+  code
+    background: #f4f4f4;
+    border: 1px solid #ddd;
+    border-left: 3px solid #f36d33;
```

Comment:
probably a good idea to use the `yellow` from https://github.com/wesbos/Syntax/blob/master/styles/_variables.styl

Reply:
That sounds great. @Wes is particular about his yellows.

Reply:
yeah he is 😅

Reply:
@416serg I got around to adding the yellow